.eleventy.js

```diff
@@ -126,21 +124,48 @@ module.exports = function(eleventyConfig) {
 
         // add functionality to group to-level items for better navigation.
         // TODO: Not currently shown in UI, but here for future use as will improve nav
```

Comment:
This comment is out of date now, right?

Reply:
right you are - will remove, thanks for spotting